test/tools/cluster.sh

```diff
@@ -0,0 +1,173 @@
+#!/bin/bash
+
+if [ "$#" -lt 1 ]; then
+    echo "usage: cluster <init|start|stop|prune|kill|uri|test> <single|replica|sharded>"
+    exit 0
+fi
+
+PORT_REPLICA=31000
+PORT_SHARDED=51000
+PORT_SINGLE=27017
+
+log () {
+  echo "$1 --------------------"
+}
+
+cluster () {
+    COMMAND="$1"
+    if [[ "$COMMAND" == "uri" ]]; then
+        TOPOLOGY=$2
+        if [[ "$TOPOLOGY" == "replica" ]] || [[ "$TOPOLOGY" == "replicaset" ]] || [[ "$TOPOLOGY" == "replica_set" ]]; then
```

Comment:
You could use a regex here:
```bash
if [[ "$TOPOLOGY" =~ ^(replica|replicaset|replica_set)$ ]]; then
```
I appreciate the flexibility of this, I would use `standalone`, `replicaset`, and `cluster` but I'm glad we can support whatever people prefer 👍

Reply:
Yes! I tried this syntax was wonky, but will try again, prob missing a character. Was about to ask for help on this, syntax is super repetitive. I often forget what the "proper" term is that the script uses. I usually go with `single`, `replica`, and `sharded` so I like this feature.

Reply:
Fixed this with your solution + functions! pretty minimal now, and easy to add / remove options.